bitmap-pegastickacorn/test/bitmap-test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+//Require in the bitmap.js file to test
+//const Bitmap = require('../model/bitmap.js');
+const bitmapFileHelper = require('/../lib/bitmap-file-helper.js');
+//Require in chai to make use of expect
+const expect = require('chai').expect;
+
+//testing the bitmap.js module
+describe('testing bitmap.js module', function(){
+  describe('testing Bitmap()', function(){
+   //testing bitmap constructor
+    it('size should equal "1365240"', function() {
+      expect(bitmapFileHelper.bitmap.size).to.equal(1365240);
```

Comment:
bitmap is not a property of bitmapFileHelper... 
try this code then write a few more test 
```
//Require in the bitmap.js file to test
//const Bitmap = require('../model/bitmap.js');
const bitmapFileHelper = require('../lib/bitmap-file-helper.js');
//Require in chai to make use of expect
const expect = require('chai').expect;

//testing the bitmap.js module
describe('testing bitmap.js module', function(){
  describe('testing Bitmap()', function(){
   //testing bitmap constructor
    it('size should equal "1365240"', function(done) {
      bitmapFileHelper.readFile('bitmap', function(err, bitmap){
        if (err) return done(err);
        expect(bitmap.size).to.equal(11078);
        done();
      });
    });
  });
});
```